### DISCUSSION TITLE
Create a public repository for all TDP dependencies (Hadoop, Spark, etc.)

### DISCUSSION THREAD

**Discussion by mehdibn** [TOSIT-IO/TDP#34] [Ideas]
# Context 
Actually, it's quite complicated to use spark, hadoop, hbase, etc. with TDP dependencies in the development part as they are not available in a public repository. The unique solution consists on building locally all TDP projects, adding to that, the dependecies between projects that make it more complex.

# Proposition
I propose to maintain a public repository with all TDP dependencies, with frequent updates after new commits to GitHub projects.

**Comment by leopaul36** [comment]
We could experiment with the Github packages which seems to support Maven: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry

I never tried this before but it looks promising and would avoid us the cost of maintaining a Nexus / Artifactory instance.

**Reply by rpignolet** [reply]
I agree to test the Github Maven registry.
